css-view-transitions-1/Overview.bs

```diff
@@ -1091,6 +1091,87 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 		To <dfn>setup view transition</dfn> given a {{ViewTransition}} |transition|,
 			perform the following steps:
 
+		Note: This algorithm captures the current state of the document,
+		calls the transition's {{UpdateCallback}},
+		then captures the new state of the document.
+
+		1. Let |document| be |transition|'s [=relevant global object's=] [=associated document=].
+
+		1. [=Capture the old state=] for |transition|.
+
+			If failure is returned, then [=skip the view transition=] for |transition| with an "{{InvalidStateError}}" {{DOMException}} in |transition|'s [=relevant Realm=],
+			and return.
```

Comment:
This used to be inline, but the algorithm was getting a bit large. This felt like a unit that could be moved to its own sub-algorithm.